Lmod-illumina.spec

```diff
@@ -49,12 +49,16 @@ BuildRequires:  lua-filesystem
 BuildRequires:  lua-json
 BuildRequires:  lua-posix
 BuildRequires:  lua-term
+%if %{rhel6_based}
+BuildRequires:  lua-bit32
+Requires:  lua-bit32
+%endif
 Requires:       lua
 Requires:       lua-filesystem
 Requires:       lua-json
 Requires:       lua-posix
 Requires:       lua-term
-Requires:       cm-config-cm = %cmrelease
+###Requires:       cm-config-cm = %cmrelease  ### unhash me when all is fine
```

Comment:
@plabrop: any ideas how to deal with the above?! either keep in a conditional or, push it away?